app/src/main/java/com/alexd23/example/myapplication/Login.kt

```diff
@@ -0,0 +1,36 @@
+package com.alexd23.example.myapplication
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.Menu
+import android.widget.Toast
+import kotlinx.android.synthetic.main.activity_login.*
+
+class Login : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setTheme(R.style.AppTheme2)
+        setContentView(R.layout.activity_login)
+        login.setOnClickListener{
+            if(et_name.text.toString().isNotEmpty() && et_surname.text.toString().isNotEmpty()) {
+                val myIntent = Intent(this, MainActivity::class.java)
```

Comment:
Intent(this, MainActivity::class.java).apply {
putExtra("name", et_name.text.toString())
}